### PR TITLE
Con 2400 20220302 user bulk md5 api

### PR DESCRIPTION
**Pull request by nbutler77**
Updates CVC/`local_dataset` functionality to check image md5s using new bulk API.

Note that this code relies on Conservator [PR 2511](https://github.com/FLIR/flirconservator/pull/2511), which implements the `check_frames_by_md5` API.

**File Changes:**

`.github/pull_request_template.md`
 - Added template for PRs

`FLIR/conservator/generated/schema.py`
 - Updated schema to bring in new `check_frames_by_md5` API

`FLIR/conservator/local_dataset.py`
 - Updates code to check image md5s in groups of 100 when uploading

`FLIR/conservator/util.py`
 - Added `chunks` function (see code for details)

[[JIRA Task](https://jiracommercial.flir.com/browse/CON-2400)]

### CODE REVIEW

File: FLIR/conservator/generated/schema.py [hunk 1]
```diff
@@ -77,6 +77,10 @@ class FavoriteAssetType(sgqlc.types.Enum):
 Float = sgqlc.types.Float
 
 
+class GraphqlID(sgqlc.types.Scalar):
+    __schema__ = schema
+
+
```
Comment (rlc2) [inline FLIR/conservator/generated/schema.py:83]: Did the generator script add this?  There are a lot of changes from ID -> GraphqlID that I don't see in Conservator's schema JSON.

Comment (rlc2) [reply FLIR/conservator/generated/schema.py:83]: Unless this schema is coming from your branch?

File: FLIR/conservator/util.py [hunk 2]
```diff
@@ -53,3 +55,16 @@ def base_convert(b, n):
...
+def chunks(list, size):
+    """
+    Splits a :param list: into lists of :param size:.
+    Returns an iterator of these lists.
+    Once the input list is exhausted, the last list in the output
+    iterator will be padded by None's to make up the size difference.
+    e.g. chunks(['a', 'b', 'c'], 2) will return an iterator containing
+    ['a', 'b'] and ['c', None]
+    Adapted from  https://stackoverflow.com/a/312644
+    """
```
Comment (rlc2) [inline FLIR/conservator/util.py:69]: Mostly, for method parameter docstrings we have followed the sphinx docstring formatting described here for the automated documentation generator: https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#template.MainClass1 -- except for the :type param: and :rtype: lines, which for conservator-cli API methods should be replaced with Python type hinting described in PEP484 (https://peps.python.org/pep-0484/).  The type hinting is new (for us), but ideally at least externally visible methods/functions should have them.

**Review by rlc2 — APPROVED** [review]

File: FLIR/conservator/util.py [hunk 3]
```diff
@@ -53,3 +55,21 @@ def base_convert(b, n):
         output.append(r)
         n = int(n / b)
     return output
+
+
+def chunks(list, size):
+    """
```
Comment (nbutler77) [inline FLIR/conservator/util.py:61]: @rlc2 Is this format correct?

Comment (rlc2) [reply FLIR/conservator/util.py:61]: It's closer, just needs a short description at the top of the docstring followed by a blank line to separate it from the :param lines.

Comment (rlc2) [reply FLIR/conservator/util.py:61]: Oh also the note starting with "once the input list is exhausted" should be between the short description and the :param lines.  So to summarize:
"""
short description

additional comments

:param X: ...
"""

**Pull request merged**